Changes

```diff
@@ -1,4 +1,8 @@
 {{$NEXT}}
+2.003006 2017-10-20
```

Comment:
Thanks for updating the changelog! Would you mind removing the added version number/date though please? That part gets updated when we push out a release.